src/reference.yaml

```diff
@@ -5736,6 +5736,970 @@ paths:
+          schema:
+            $ref: '#/definitions/RateLimitError'
+          headers:
+            X-Request-Id:
+              type: integer
+              description: A unique reference for the request.
+        '500':
+          description: Internal server error
+          x-errors:
+            - code: internal_server_error
+              category: base
+              message: Internal server error
+              params: ''
+          headers:
+            X-Request-Id:
+              type: integer
+              description: A unique reference for the request.
+        '503':
+          description: Service is temporary unavailable
+          x-errors:
+            - code: service_unavailable
+              category: base
+              message: Service is temporary unavailable
+              params: ''
+          headers:
+            X-Request-Id:
+              type: integer
+              description: A unique reference for the request.
+        default:
+          description: Unexpected error.
+          headers:
+            X-Request-Id:
+              type: integer
+              description: A unique reference for the request.
+  /conversions/{id}/date_change:
+    post:
+      tags:
+        - Conversions
+      x-api-group: convert
+      summary: Cancel Conversion
+      description: 'Change the settlement date of a conversion'
+      operationId: ConversionDateChange
+      consumes:
+        - multipart/form-data
+      produces:
+        - application/json
+      parameters:
+        - name: X-Auth-Token
+          in: header
+          required: true
+          type: string
+          description: Authentication token.
+        - name: id
+          in: path
+          required: true
+          type: string
+          description: ID of the conversion that is being changed
+        - name: on_behalf_of
+          in: formData
+          required: false
+          type: string
+          description: Contact UUID.
+        - name: new_settlement_date
+          required: false
+          type: string
+          in: formData
+          description: New conversion settlement date
+      responses:
+        '200':
+          description: Success.
+          schema:
+            $ref: '#/definitions/ConversionDateChange'
+          headers:
+            X-Request-Id:
+              type: integer
+              description: A unique reference for the request.
+        '400':
+          description: Client error.
+          x-errors:
+            - code: id_is_not_valid_uuid
+              category: id
+              message: id should be in UUID format
+              params: ''
+            - code: on_behalf_of_self
+              category: on_behalf_of
+              message: You cannot act on behalf of your own Contact
+              params: ''
+            - code: contact_not_found
+              category: on_behalf_of
+              message: Contact was not found for this id
+              params: ''
```

Comment:
Believe we also want these additional errors:
```
            - code: invalid_extra_parameters
              category: base
              message: Invalid extra parameters extra_parameter
              params: '{ "parameters" => "extra_parameter" }'
            - code: new_settlement_date_invalid_in_past
              category: base
              message: New settlement date has to be today or greater than today's date
              params: ''
            - code: new_settlement_date_invalid
              category: base
              message: Invalid new settlement date, next valid date time is - YYYY-MM-DDT15:30:00Z
              params: ''
```

Reply:
Done